File: ios/mEEting/GoModule.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(MyGoModule, NSObject)

RCT_EXTERN_METHOD(exampleFunction:(NSString *)name resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)

@end

File: ios/mEEting/GoModule.swift
import Foundation

@objc(MyGoModule)
class MyGoModule: NSObject {
  @objc
  func exampleFunction(_ name: String, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) {
    let result = BackendsomeFunction(name)
    resolve(result)
  }

  // React Native requires this method to be present
  @objc static func requiresMainQueueSetup() -> Bool {
    return true
  }
}